### PR TITLE
Adding Albert Hidalgo

### PR DESCRIPTION
**Pull request by itsalb3rt**

### CODE REVIEW

File: DominicanWhoCodes.Blazor/wwwroot/data/developers.json [hunk 1]
```diff
@@ -134,6 +134,15 @@
     "linkedin": "www.linkedin.com/in/AVTO",
     "github": "https://github.com/Avto9711",
     "twitter":"https://twitter.com/angeliisimo"
+  },
+  {
+    "name": "Albert Eduardo Hidalgo Taveras",
+    "initials": "AEHT",
+    "image": "data/devs-pics/albert-hidalgo.jpg",
+    "summary": "Super moto and coding",
+    "skills": "PHP (Symfony, Ligne[Own Framework]), Vue, UX",
```
Comment (AngelGarcia13) [inline DominicanWhoCodes.Blazor/wwwroot/data/developers.json]: Check this out, I think you missed a ")" after "(Symphony".

Fix this and resolve the conflicts that will generate the merge I'm about to merge (#40) from @cvivieca

**Comment by itsalb3rt** [comment]
check now, everything is right

**Comment by AngelGarcia13** [comment]
> check now, everything is right

Now I'm having conflicts with the #40 and I just revert this PR.

Can you please resolve again, my bad, you are the next in the queue

**Comment by itsalb3rt** [comment]
No problem, thanks! :smiley:

**Comment by AngelGarcia13** [comment]
@itsalb3rt shipped!

**Pull request merged**